lib/rails_util/timezone_helper.rb

```diff
@@ -14,42 +15,29 @@ class << self
       # @param [Symbol=>[Date]] options key-value option pairs, used to provide a separate Date object if a specific date is needed, otherwise the date of the time argument is used
       # @return DateTime object with the time in the desired `to_timezone`, and a date of either the provided date or time
       def convert_timezone(time, to_timezone, from_timezone, **options)
-        date = options.fetch(:date, time)
-        utc_time = convert_to_utc(time, from_timezone, date)
-        convert_from_utc(utc_time, to_timezone, date)
+        converted_time = time + offset_in_seconds(time, to_timezone, from_timezone)
```

Comment:
@bhennes2  was able to highly simplify with your suggestion!

Added a spec to check out what happens during the other half of the year

Reply:
Nice!